tasks/gulp/clean.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const gulp = require('gulp')
+const taskArguments = require('./task-arguments')
+const del = require('del')
+
+// Clean task for a specified folder --------------------
+// Removes all old files, except for package.json
+// and README in all package
+// '/**/**/*' is to remove folder deep down  in globals/
```

Comment:
I'm also not sure that `${destination}/**/**/*` is needed - as `**` allows for any numbers of directories so folders within globals should be included as part of `${destination}/**/*.*`? It definitely feels like if we get the syntax right we should only need one 'positive' path in the list.

Reply:
i can have a look, i found that **/* didnt delete folders 3 levels down packages/globals/var for example

Reply:
Is that because `**/*.*` should be `**/*`?

Reply:
So I think the issue is that `**/*` deletes component folders even if they still contain a package.json file?